style.css

```diff
@@ -394,6 +404,205 @@ h1 {
 }
 
 footer {
+  align-self: center;
+  justify-content: center;
   background: url("img/Frame.png") bottom 50px right no-repeat;
   background-size: 15%;
 }
+
+@media (min-width: 600px) {
+  header {
+    background: url("img/header-illsutration-mobile.png") center center no-repeat;
+    background-size: cover;
+  }
+
+  #header-welcome {
+    background: none;
+  }
+
+  .footer-icons ul {
+    justify-content: center;
+  }
+
+  footer {
+    background-size: 10%;
+  }
+}
+
+@media (min-width: 768px) {
+  header {
+    background: url("img/header-illsutration-mobile.png") center center no-repeat;
+    background-size: cover;
+    height: 1150px;
+  }
+
+  #header-welcome {
+    margin-top: 10rem;
+  }
+
+  #more-text {
+    margin-top: 5rem;
+  }
+
+  #header-welcome .social-icons {
+    margin-top: 7rem;
+  }
+}
+
+@media (min-width: 992px) {
+  #more-text {
+    width: 60%;
+  }
+
+  body {
+    margin: 0;
+    width: 100vw;
+  }
+
+  #welcome-text {
+    display: none;
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/134432768-84fcdc34-9f9b-4a10-b8c2-9953a412ef44.png)

- [ ] As you can see from the picture for desktop view (screen-width 992px and higher) The horizontal scroll bar is showing which means that there's an overflow. Kindly fix the issue.


![image](https://user-images.githubusercontent.com/43698511/134433519-a02457a7-c424-4d60-a865-f51cf385b399.png)

- [ ] In the original design The Title My recent work is followed by 2 rows with 3 projects each. Your webpage looks a bit different for screen-width from 992px to 1199px kindly fix the issue.